test/cli/test-fixtures/image-hidden-packages/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7.9.2009@sha256:dead07b4d8ed7e29e98de0f4504d87e8880d4347859d839686a31da35a3b532f
```

Comment:
this image was so old that I was having a hard time replicating the state, so I updated the test fixture + test